transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryFactory.java

```diff
@@ -20,10 +20,12 @@
     private static TransferMethodRepositoryFactory sInstance;
     private TransferMethodRepository mTransferMethodRepository;
     private TransferMethodConfigurationRepository mTransferMethodConfigurationRepository;
+    private TransferMethodUpdateConfigurationRepository mUpdateTransferMethodConfigurationRepository;
```

Comment:
`mUpdateTransferMethodConfigurationRepository` should be `mTransferMethodUpdateConfigurationRepository`

Reply:
fixed